src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/LongOperationStart/LongOperationInDataViewDelegateTests.cs

```diff
@@ -6,13 +6,14 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using Xunit;
 using System.Threading.Tasks;
+using Acuminator.Utilities;
 
 namespace Acuminator.Tests.Tests.StaticAnalysis.LongOperationStart
 {
     public class LongOperationInDataViewDelegateTests : DiagnosticVerifier
     {
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
-            new PXGraphAnalyzer(new LongOperationInDataViewDelegateAnalyzer());
+            new PXGraphAnalyzer(CodeAnalysisSettings.Default, new LongOperationInDataViewDelegateAnalyzer());
```

Comment:
Don't you need `WithRecursiveAnalysisEnabled()` here? Tests should not rely on the default settings because they can change in the future; all settings that are used in the tests must be set explicitly.

Reply:
Ok